src/main/java/io/mfj/expr/ExprLogicStatement.kt

```diff
@@ -154,7 +154,13 @@ class ExprLogicStatement(val left: ExValue, var op: ExLogicOpType, val right:ExV
           }
         }
       }
-      ExLogicOpType.NOT_EQUAL -> { leftVal != rightVal }
+      ExLogicOpType.NOT_EQUAL -> {
+        if ( leftVal is Number && rightVal is Number ) {
```

Comment:
what do you think of doing this recursively-ish, like:
```
ExLogicOpType.NOT_EQUAL -> {
    !calc(leftVal, ExLogicOpType.EQUAL, rightVal)
}
```

This would have the advantage of not missing any bug fixes or changes to EQUAL, and I'm pretty sure the logic inverts cleanly.